tests/utils/test_env_checker.py

```diff
@@ -192,6 +194,31 @@ def test_check_reset_return_info_deprecation(test, func: callable, message: str)
         check_reset_return_info_deprecation(GenericTestEnv(reset_fn=func))
 
 
+def test_check_seed_deprecation():
+    """Tests that return_info has been correct deprecated as an argument to `env.reset()`."""
+
+    message = """Official support for the `seed` function is dropped. Standard practice is to reset gym environments using `env.reset(seed=<desired seed>)`"""
+
+    env = GenericTestEnv()
+
+    def seed(seed):
+        return
+
+    with pytest.warns(
+        UserWarning, match=f"^\\x1b\\[33mWARN: {re.escape(message)}\\x1b\\[0m$"
+    ):
+        env.seed = seed
+        assert callable(env.seed)
+        check_seed_deprecation(env)
+
+    with warnings.catch_warnings():
```

Comment:
To check this works as intended we might want to add 
```
with warnings.catch_warnings(record=True) as caught_warnings:
    <test code>

assert len(caught_warnings) == 0

```

Reply:
sounds good, I changed to this pattern